data/35e/wizards_of_the_coast/campaign_settings/dragonlance/campaign/dcs_profs_weapon.lst

```diff
@@ -1,2 +1,10 @@
+# CVS $Revision: $ $Author: $ -- Thu Jul 10 13:59:32 2014 -- reformated by prettylst.pl v1.51 (build 24365)
+<<<<<<< HEAD
+Shadowstaff			TYPE:Exotic.Melee.ExoticMelee.TwoHanded.TwoHandedMelee.TwoHandedExoticMelee
+=======
 
-Shadowstaff		TYPE:Exotic
+# Misc for PCGen
+# Weapon Name                   Type
+
+Flail (Dire).COPY=Shadowstaff
+>>>>>>> efe2235fa822c96a14bbd2f7c7e2908631cac3c9
```

Comment:
And this file conflict. It is best to resolve conflicts before submitting, these will cause errors.